### ISSUE TITLE
[Package Request]: AMD Chipset Driver

### ISSUE THREAD

**Issue by Barradel** [microsoft/winget-pkgs#98416]
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not a script file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: AMD
Publisher Url: https://www.amd.com/en/support/kb/release-notes/rn-ryzen-chipset-5-02-19-2221
Package Name: AMD Ryzen Chipset Driver
Description: AMD Chipset Drivers, Power Plans and Power Provisioning Package
Package Version: 5.02.19.2221
Installer URL: https://drivers.amd.com/drivers/amd_chipset_software_5.02.19.2221.exe

Current Package Identifier (for new versions):

**Comment by licon4812** [comment]
![image](https://user-images.githubusercontent.com/32421608/222901645-fe74dc79-7050-41e5-a32d-95cd604d42db.png)

**Comment by MagicAndre1981** [comment]
you need to pass a referer to download AMD drivers via script:

`wget.exe --referer=https://www.amd.com/en/support/chipsets/amd-socket-am4/x570 https://drivers.amd.com/drivers/amd_chipset_software_5.02.19.2221.exe`

I'm not sure how winget currently can handle this.

**Comment by Barradel** [comment]
I am unsure of the appropriate steps to take. If you first navigate to https://www.amd.com/en/support/kb/release-notes/rn-ryzen-chipset-5-02-19-2221, the link https://drivers.amd.com/drivers/amd_chipset_software_5.02.19.2221.exe will function properly.

**Comment by MagicAndre1981** [comment]
yes, in this case the referer is set by the browser, when you click the url directly not.

**Comment by Karl-WE** [comment]
as download link we can rely on computerbase.de, they are updating their repo frequently and ads free.

https://media.computerbase.de/s/WzznBR5aYGZ_75BUccAb_A/1690838826/download/412/amd_chipset_software_5.05.16.529.exe

**Comment by steffenweber** [comment]
These links expire after 1 hour, sorry.

**Comment by Karl-WE** [comment]
alright. dang :|

**Comment by Macleykun** [comment]
> you need to pass a referer to download AMD drivers via script:
> 
> `wget.exe --referer=https://www.amd.com/en/support/chipsets/amd-socket-am4/x570 https://drivers.amd.com/drivers/amd_chipset_software_5.02.19.2221.exe`
> 
> I'm not sure how winget can handle this.

May i ask how you got this to work on your end? I tried this on my end but couldn't find wget.exe, using wget as is on windows seems to not work as it doesn't understand the referer argument.

Are you using a different wget?

**Comment by MagicAndre1981** [comment]
ok, great you got it working, I use the **JernejSimoncic.Wget** in version 1.21.4

**Comment by DandelionSprout** [comment]
I'm currently getting this when trying to create a manifest of 7.06.02.123:

![Image](https://github.com/user-attachments/assets/7eaf81ea-3b38-4ffe-9f75-1de93034ba84)

So essentially it's still proving overly hard to create a manifest of it several years later.
